Animations/Zoom.js

```diff
@@ -1,25 +1,25 @@
 var makeAnimation = require('../src/TuxAnimations');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.